src/PowerShellEditorServices.Protocol/LanguageServer/Completion.cs

```diff
@@ -58,6 +58,12 @@ public enum CompletionItemKind
         Folder = 19
     }
 
+    public enum InsertTextFormat
+    {
+        PlainText = 1,
```

Comment:
Does this have to be 1 based?

Reply:
Well, that is what LSP expects and in general, we've also mapped 1:1 between our enum values and what LSP expects.

https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.languageserver.protocol.inserttextformat?view=visualstudiosdk-2017
https://developercommunity.visualstudio.com/content/problem/258457/lsp-hangs-on-completions-if-inserttextformat-is-no.html